moxy-compiler/src/main/java/moxy/compiler/viewstate/ViewInterfaceProcessor.kt

```diff
@@ -236,13 +237,42 @@ class ViewInterfaceProcessor(
         return modifiers.contains(Modifier.STATIC)
     }
 
-    private inline fun <T, R> List<T>.flatMapToSet(transform: (T) -> Iterable<R>): Set<R> {
-        return flatMapTo(LinkedHashSet(), transform)
-    }
-
     companion object {
         private const val OPTION_DEFAULT_STRATEGY = MvpCompiler.DEFAULT_MOXY_STRATEGY
         private val DEFAULT_STATE_STRATEGY: TypeElement =
             elementUtils.getTypeElement(AddToEndSingleStrategy::class.java.canonicalName)
     }
+
+    class StockViewMethod(
```

Comment:
You should add kdoc comments to classes/methods, whose purpose is unclear from first sight (like for StockViewMethod)